processor/src/operations/u32_ops.rs

```diff
@@ -188,6 +186,9 @@ impl Process {
 
         self.stack.set(0, result);
         self.stack.shift_left(2);
+        self.decoder
+            .set_user_op_helpers(Operation::U32or, &[self.bitwise.last_cycle_start_row()]);
+
```

Comment:
Instead of making another call to the bitwise co-processor to get the row address, could be be returned (in addition to the result) from the initial call to compute the value: `self.bitwise.u32or(a, b)?;`?

Reply:
I considered this. I'm happy to change it.